### DISCUSSION TITLE
class-validator or not?

### DISCUSSION THREAD

**Discussion by TamirCode** [omar-dulaimi/prisma-class-validator-generator#7] [Q&A] [answered]
can u screenshot one of the generated DTOs in the readme , also where do you put the actual class-validators such as @MinLength or @isEmail ? it would be nice if you put them in your schema.prisma and then they would generate DTOs with class-validators, but it isnt so clear what your package does and your screenshots have nothing to do with class validators, so pls get back to me thanks

**Comment by omar-dulaimi** [comment, marked as answer]
@TamirCode I have updated the README, check it out.

Also passing validators through schema is one thing I would like to do in all of my generators. But I have't had a chance yet.

You can submit a PR if you would like.
